qiskit/extensions/__init__.py

```diff
@@ -61,10 +61,9 @@
 )
 from .simulator import Snapshot
 
-
 warnings.warn(
-    "The qiskit.extensions module is pending deprecation since Qiskit 0.45.0. It will be deprecated "
-    "in a following release, no sooner than 3 months after the 0.45.0 release.",
+    "The qiskit.extensions module is deprecated since Qiskit 0.46.0. It will be deprecated "
+    "in a following release, no sooner than 3 months after the 0.46.0 release.",
```

Comment:
We actually have a solid date for this, so we could update it to say "will be removed in Qiskit 1.0" if you like?

Reply:
Good call, done!